src/main/java/DataModel/ExchangeRate.java

```diff
@@ -1,12 +1,41 @@
 package DataModel;
 
+import Mappers.JSONMapper;
+import services.FetchToTheForexAPI;
+
+import java.util.HashMap;
 import java.util.Objects;
 
 public class ExchangeRate {
     private String ticker;
     private String date;
     private double exchangeAmount;
-    private double rate = 4.2;
+    FetchToTheForexAPI fetch = new FetchToTheForexAPI();
```

Comment:
Why do you use methods in the model?
Is it better to separate it?

Reply:
I I didn't know it had to be separated. I'll try to do it.